src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/camunda/CamundaReconfigureInputVariableDefinition.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.camunda;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.BusinessContext;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.CFTTaskState;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.TaskSystem;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.ExecutionTypeResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.NoteResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.TaskRoleResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.WorkTypeResource;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@ToString
+@EqualsAndHashCode
+@Getter
+@Setter
```

Comment:
Please keep objects immutable and avoid setters unless absolutely necessary.  You can provide constructors for creation.

Reply:
updated